environs/bootstrap/bootstrap_test.go

```diff
@@ -122,6 +122,24 @@ func (s *bootstrapSuite) TestBootstrapSpecifiedConstraints(c *gc.C) {
 	c.Assert(env.args.ModelConstraints, gc.DeepEquals, modelCons)
 }
 
+func (s *bootstrapSuite) TestBootstrapSpecifiedBootstrapSeries(c *gc.C) {
+	env := newEnviron("foo", useDefaultKeys, nil)
+	s.setDummyStorage(c, env)
+	cfg, err := env.Config().Apply(map[string]interface{}{
+		"default-series": "wily",
+	})
+	c.Assert(err, jc.ErrorIsNil)
+	env.cfg = cfg
+
+	err = bootstrap.Bootstrap(envtesting.BootstrapContext(c), env, bootstrap.BootstrapParams{
+		BootstrapSeries: "trusty",
+	})
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(env.bootstrapCount, gc.Equals, 1)
+	c.Assert(env.args.BootstrapSeries, gc.Equals, "trusty")
+	c.Assert(env.args.AvailableTools.AllSeries(), jc.SameContents, []string{"trusty"})
```

Comment:
Change the prior 3 asserts to `.Check` so that we get all errors in a failing test. `.Assert` needs to be kept on the err as a gate.